### DISCUSSION TITLE
検索結果のリストアップ画面から「コメント」数表示が消えた

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#419] [要望・アイデア・不具合]
本日、記事を検索したところ、昨日までは表示されていた「コメント」の数が、今では表示されないようになってるようです。「コメント」の数も表示されるように、以前の状態に戻していただけませんか。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご報告ありがとうございます🙏
検索ページのアップデートを行った際に、誤って表示されなくなってしまったようです
元に戻しますので、しばらくお待ちいただけますと幸いです🙇

**Reply by tsubolabo** [reply]
@tanaka-asami415 

了解しました。
しかし、以前よりも改良されている部分もありますので、これらはそのままでお願いします。
+ ストック数も表示されるようになって情報量が増えた。以前はいいね数とコメント数だけだった。
+ 表示位置が左下になって見やすい。以前は右上なので、視線を大きく移動しなければならず快適ではなかった。

**Reply by tsubolabo** [reply]
その場で思いついたままの小刻みのコメントばかりで失礼します。新フォーマットについて、その後に感じた点を追記します。

　以前に比べて縦方向のスペース資源を消費しすぎていて、スクロールしながら閲覧するのが怠くなった気がします。また、タイトルとタグ名はかなり密接な関係にあるので、記事内容を素早く把握するためにはあまり離れていない方が便利です。

そこで、レイアウトを次のように変えることを提案します。

現在：
　投稿者ID
　日付
　記事タイトル
　記事内容の抜粋
　タグ名の横並び
　いいね、ストックマークの横並び

提案：
　投稿者ID
　日付、いいね、ストック『、コメント』マークの横並び
　記事タイトル
　タグ名の横並び
　記事内容の抜粋

「記事内容の抜粋」の行間も以前より広がっていませんか？
（以上、パソコン版限定での感想です。今のところ、スマホはあまり見ていません。）

**Reply by tanaka-asami415** [reply]
@tsubolabo 
お待たせいたしました！コメント数も表示されるように修正が完了しました。
追加でいただいたご意見は、今後の改善の参考にさせていただきます🙇

**Reply by tsubolabo** [reply]
@tanaka-asami415 
> コメント数も表示されるように修正が完了しました。

確認できました。どうもありがとうございました。
> 追加でいただいたご意見は、今後の改善の参考にさせていただきます

賛同者は沢山おられると思います（勝手な想像）。宜しくお願いします。

**Reply by tsubolabo** [reply]
さらなる要望です。

「タグ名の横並び」はタイトルと一緒に目に入れて、記事の内容を即座に把握するために利用します。しかし、灰色なのでほとんど目立たず、他の情報との識別が困難です。もっと目立つように濃い色でお願いします。
見た所、色を使わないでモノクロ調に統一することに拘っているような感じを受けますが、できれば真黒よりも、緑色などにすると、タイトルとの見分けもつきやすくなると思います。
